js/this-inside-foo-bar/post.md

```diff
@@ -0,0 +1,81 @@
+# Title - What is `this` inside foo.bar()? 
+
+## The problem
+
+![Code showing - method which is passed as callback loses reference to original obj as this, but calling obj.method() directly works](https://cdn.hashnode.com/res/hashnode/image/upload/v1584584613992/rwNrFCElj.png)
```

Comment:
In the image it should be `onClick` instead of `.onClick` and it would be better if its called in next line

Reply:
Makes sense. Changed image and code to make evt handler usage and implementation clearer.